blobstore/src/test/java/org/jclouds/blobstore/integration/internal/BaseContainerIntegrationTest.java

```diff
@@ -595,6 +596,32 @@ public void testListMarkerPrefix() throws Exception {
       }
    }
 
+   /** Test that listing with an empty string for prefix and delimiter returns all of the keys. */
+   @Test(groups = {"integration", "live"})
+   public void testListEmptyPrefixDelimiter() throws Exception {
+      final String container = getContainerName();
+      BlobStore blobStore = view.getBlobStore();
+
+      try {
+         blobStore.createContainerInLocation(null, container);
```

Comment:
This should be outside the `try` block so that a failed call to `createContainerInLocation` does not return a bogus container in the `finally` block.